server/channels/app/platform/utils.go

```diff
@@ -20,3 +20,14 @@ func maxInt(a, b int) int {
 	}
 	return b
 }
+
+// getCacheTargets is used to fill target value types
+// for getting items from cache.
+func getCacheTargets[T any](l int) []any {
+       toPass := make([]any, 0, l)
+       for i := 0; i < l; i++ {
+               var target T
+               toPass = append(toPass, &target)
```

Comment:
Apologies if this is a known gotcha, but can't we simply append `new(T)`?

Reply:
Should be possible. I'll confirm.